src/main/java/com/ridemart/enums/Make.java

```diff
@@ -0,0 +1,47 @@
+package com.ridemart.enums;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public enum Make {
```

Comment:
Using Enums is great but here, when it comes to Make and Model, it will create some difficulties in matter of supporting the application. Every time you need to remove or add something, you need a new commit.
I would recommend after you are done with all the other changes ADN after you have added unit tests for your Controllers and Service classes, to consider extracting AT LEAST Make and Model as their own tables in the database. It would be ideal to have all 4 enums extracted as tables, but Make and Model are the most important!